<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HFlatStyleWidget</class>
 <widget class="QWidget" name="HFlatStyleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <item row="3" column="2" colspan="2">
    <widget class="QSlider" name="horizontalSlider_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QSlider" name="horizontalSlider_1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QProgressBar" name="progressBar_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>16</height>
      </size>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar_1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>16</height>
      </size>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>测试按钮</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton_1">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>测试按钮</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>测试按钮</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit_1"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButton_4">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>测试按钮</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="4" column="0">
    <widget class="QRadioButton" name="radioButton_1">
     <property name="text">
      <string>语文</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QRadioButton" name="radioButton_2">
     <property name="text">
      <string>英语</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QRadioButton" name="radioButton_3">
     <property name="text">
      <string>数学</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QRadioButton" name="radioButton_4">
     <property name="text">
      <string>历史</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="checkBox_1">
     <property name="text">
      <string>选项1</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>选项2</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>选项3</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QCheckBox" name="checkBox_4">
     <property name="text">
      <string>选项4</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="6">
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="gridStyle">
      <enum>Qt::DashDotLine</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="6">
    <widget class="QSlider" name="verticalSlider_1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QScrollBar" name="horizontalScrollBar_1">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="5" rowspan="6">
    <widget class="QScrollBar" name="verticalScrollBar_1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBar_1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBar_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
